lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1286,9 +1286,10 @@ export class UserAgentApplication {
      */
     private renewToken(scopes: Array<string>, resolve: Function, reject: Function, account: Account, serverAuthenticationRequest: ServerRequestParameters): void {
         const scope = scopes.join(" ").toLowerCase();
-        this.logger.verbose("renewToken is called for scope:" + scope);
+        this.logger.verbose("renewToken is called for scope: " + scope + " authority: " + serverAuthenticationRequest.authority);
+
+        const frameName = `msalRenewFrame${Constants.resourceDelimiter}${scope}${Constants.resourceDelimiter}${serverAuthenticationRequest.authority}`;
```

Comment:
Since this is doing essentially the same thing in two places, I think this should be factored into a helper function (in WindowUtils?)

```ts
function generateFrameName(prefix: string, scopes: string[], authority: string): string {
  return `${prefix}${Constants.resourceDelimiter}${scopes.join(" ").toLowerCase()}${Constants.resourceDelimiter}${authority}`;
}
```

I would also like to see tests included.

Reply:
Tests are tricky, I am trying to check how I can test an iframe generated, Will get back on this.

Reply:
I'm specifically referring to testing the factored out helper function that would generate the name.

Reply:
Ah! That is straightforward and I think it is enough test for this. Will update this PR soon.

Reply:
Done